### PR TITLE
66-feature-for-update

### PR DESCRIPTION
**Pull request by ElioZhou**

### CODE REVIEW

**Review by ElioZhou — COMMENTED** [review]
update

**Review by ElioZhou — COMMENTED** [review]
update

File: service/static/js/rest_api.js [hunk 1]
```diff
@@ -6,13 +6,13 @@ $(function () {
...
-        $("#product_description").val(res.description);
-        $("#product_price").val(res.price);
+        $("#product_id_for_update").val(res.id);
+        $("#product_name_for_update").val(res.name);
+        $("#product_category_for_update").val(res.category);
+        $("#product_description_for_update").val(res.description);
+        $("#product_price_for_update").val(res.price);
         
-        $("#product_like").val(0);
+        $("#product_like_for_update").val(0);
```
Comment (marfa0414) [inline service/static/js/rest_api.js:15]: we don't have like in update form

File: service/static/js/rest_api.js [hunk 2]
```diff
@@ -108,26 +108,26 @@ $(function () {
...
-        let category = $("#pet_category").val();
-        let available = $("#pet_available").val() == "true";
-        let gender = $("#pet_gender").val();
-        let birthday = $("#pet_birthday").val();
+        let product_id = $("#product_id_for_update").val();
+        let name = $("#product_name_for_update").val();
+        let category = $("#product_category_for_update").val();
+        let description = $("#product_description_for_update").val();
+        let price = $("#product_price_for_update").val();
+        let like = $("#product_like_for_update").val();
```
Comment (marfa0414) [inline service/static/js/rest_api.js:116]: same

**Pull request merged**